content/pages/news/media21-prostate-mri-cspca-cad

```diff
@@ -0,0 +1,11 @@
+title: Study on End-to-end Automated 3D Prostate Cancer Detection in MRI published in Medical Image Analysis
+date: 2021-07-04
+description: Anindo Saha and Matin Hosseinzadeh, led by Henkjan Huisman, investigate state-of-the-art practices across recent literature and harmonize their findings into an automated 3D detection/diagnosis model for clinically significant prostate cancer in MRI. 
+picture: news/media21-prostate-mri-cspca-cad.png
+groups: diag
+
+Prostate cancer is one of the most prevalent cancers in men worldwide. In the absence of experienced radiologists, its multifocality, morphological heterogeneity and strong resemblance to numerous non-malignant conditions in MR imaging, can lead to low inter-reader agreement (<50%) and sub-optimal interpretation. The development of automated, reliable detection algorithms has therefore become an important research focus in medical image computing, offering the potential to support radiologists with consistent quantitative analysis in order to improve their diagnostic accuracy, and in turn, minimize unnecessary biopsies for patients.
+
+Published in [*Medical Image Analysis*](https://www.journals.elsevier.com/medical-image-analysis), **‘End-to-end Prostate Cancer Detection in bpMRI via 3D CNNs: Effects of Attention Mechanisms, Clinical Priori and Decoupled False Positive Reduction’** explores several state-of-the-art techniques from recent literature and harmonizes their findings into a novel 3D detection/diagnosis AI model, which can automatically localize clinically significant prostate cancer in bi-parametric MRI. Notably, in this study, [Anindo Saha](https://www.diagnijmegen.nl/people/anindo-saha/) and [Matin Hosseinzadeh](https://www.diagnijmegen.nl/people/matin-hosseinzadeh/), led by [Henkjan Huisman](https://www.diagnijmegen.nl/people/henkjan-huisman/), demonstrate that using a large dataset of 1950 prostate MRI paired with radiologically-estimated (PI-RADS v2) annotations, modern 3D convolutional neural networks can be trained to detect histologically-confirmed (Gleason Grade Group > 1) malignancies in an external cohort —demonstrating strong generalization by reaching moderate agreement with expert radiologists and independent pathologists. Furthermore, the highly diverse patient cohort investigated in this study covers the complete distribution of MRI exams encountered during clinical routine and is one of the largest of its kind used in prostate-AI research, thus far.
```

Comment:
You should hyperlink your names with [member/anindo-saha] and [member/henkjan-huisman], for example. Check https://github.com/DIAGNijmegen/website-content/blob/master/content/pages/news/pulmonary-nodule-malignancy-risk-radiology-2021.md for how I have hyperlinked my name and Colin's name.

Reply:
Done.